pkg/api/v1/routes/handlers.go

```diff
@@ -125,17 +125,6 @@ func (r *Routes) serverEnroll(c *gin.Context) (int, *v1types.ServerResponse) {
 	span.SetAttributes(attribute.KeyValue{Key: "serverId", Value: attribute.StringValue(id)})
```

Comment:
Probably should move this down to around line 151, so we don't have multiple attributes for `server id` on this span.